IntegrationTests/ReducedConfig/IRtoTB/script/compileHLS.tcl

```diff
@@ -0,0 +1,100 @@
+# Common settings
+source ../../../../project/env_hls.tcl
+file link -symbolic common ../../../common
```

Comment:
If I want to recompile the HLS, the tcl script complains about not being able to create new links until I've manually removed them. Could this be automised so it's fool-proof? (I'm lazy and stupid; managed several times to delete the actual files and not just the link...)

```Sourcing Tcl script 'compileHLS.tcl'
could not create new link "common": that path already exists
    while executing
"file link -symbolic common ../../../common"
    (file "compileHLS.tcl" line 3)
    invoked from within
"source compileHLS.tcl"
    ("uplevel" body line 1)
    invoked from within
"uplevel \#0 [list source $arg] "
```

Reply:
I also wanted this to be able to create broken symlinks, which is useful for somebody running this interactively before emData/download.sh has been run. So I replaced "file link" with a function that just executes the "ln" shell command. This takes care of both issues.